src/core/selinux-setup.c

```diff
@@ -104,3 +107,38 @@ int mac_selinux_setup(bool *loaded_policy) {
 
         return 0;
 }
+
+int mac_selinux_setup_namespaced_policy(const char *policy) {
```

Comment:
This can just be placed in `exec-invoke`? It's for spawned processes only?

Reply:
Yes